src/js/main.js

```diff
@@ -0,0 +1,213 @@
+// import smoothScroll from 'smoothScroll';
```

Comment:
@PalleZingmark I can't seem to get import to work - can you see what I am doing wrong? There is a smoothScroll.js

Reply:
Yes, you're compiling ES6 Modules to [CommonJS format when using Babel](https://babeljs.io/learn-es2015/#ecmascript-2015-features-modules) so you either need to use Browserify or WebPack to bundle your modules.

Reply:
Ah, ok. Since Grunt is in use maybe Browserify is a good pick. WebPack might be overkill and better suited for a next project, yes? I'll take a look during the weekend!

Reply:
Well, Webpack isn't a task runner and actually go quite well hand in hand with e.g. Grunt. 

...aaaand, ehum. I think it's better that we focus on getting the code to flow, than getting stuck on tooling,  so I actually took the liberty and added Webpack, so imports work now. 
We can go thru my changes in details next time. :)